src/tests/ftest/cart/utest/utest_portnumber.c

```diff
@@ -193,7 +193,7 @@ static void
 test_port_tcp(void **state)
 {
 	d_setenv("OFI_INTERFACE", "lo", 1);
-	d_setenv("CRT_PHY_ADDR_STR", "ofi+tcp;ofi_rxm", 1);
```

Comment:
if changing this file again would be good to update to use D_INTERFACE and D_PROVIDER instead